api/src/comment/guard/comment-auth.guard.ts

```diff
@@ -1,19 +1,22 @@
 import { CanActivate, ExecutionContext, Injectable } from '@nestjs/common';
-import { ListDao } from '../../list/dao/list.dao';
+import { CommentsService } from '../comment.service';
 
 @Injectable()
 export class CommentAuthorizationGuard implements CanActivate {
-  constructor(private listsDao: ListDao) {}
+  constructor(private commentService: CommentsService) {}
 
   async canActivate(context: ExecutionContext) {
     const request = context.switchToHttp().getRequest();
     const user = request.user;
-    const listId = request.params.id || request.body.listId;
     const commentId = request.param.commentId || request.body.commentId;
```

Comment:
This is a typo from some moron. Plus since we never send it in body anymore it can be changed to:
```suggestion
    const commentId = request.params.commentId;
```